ui/plugin-system/src/model/variables.ts

```diff
@@ -12,12 +12,12 @@
 // limitations under the License.
 
 import { ListVariableDefinition } from '@perses-dev/core';
-import { LegacyDatasources } from '../runtime';
+import { DatasourcesContextType } from '../runtime';
 
 export type VariableOption = { label: string; value: string };
 
 export interface GetVariableOptionsContext {
-  datasources: LegacyDatasources;
+  datasources: DatasourcesContextType;
```

Comment:
Suggestion: The plural `datasources` makes the property sound like an array of objects. I know we don't really like the name `datasourceSrv` but would something like`datasourceManager` or `datasourceStore` be a better name to denote its an object with methods on it? Feel free to ignore but something I was just thinking about as I read this.

Reply:
Oh man, I hate the name too and was thinking the same thing. 😂  I'll think about it and try and come up with something better.